openpype/plugins/publish/collect_slate_global.py

```diff
@@ -0,0 +1,121 @@
+import os
+import json
+import pyblish.api
+from openpype import resources
+
+
+class CollectSlateGlobal(pyblish.api.InstancePlugin):
+    """Inject the data needed to generate Slates in the enabled families."""
+    label = "Collect for Slate Global workflow"
+    order = pyblish.api.CollectorOrder + 0.499
+    families = [
+        "review",
+        "render"
+    ]
+
+    def process(self, instance):
+
+        context = instance.context
+        slate_settings = context.data["project_settings"]["global"]\
+            ["publish"].get("ExtractSlateGlobal")
+
+        if not slate_settings:
+            self.log.warning("No slate settings found. Skipping.")
+            return
+
+        if not slate_settings["enabled"]:
+            self.log.warning("ExtractSlateGlobal is not active. Skipping.")
+            return
+
+        if context.data.get("host") == "nuke" and \
+                "render.farm" in instance.data.get("families"):
+            self.log.warning(
+                "Skipping Slate Global Collect in Nuke context, defer to "
+                "Deadline."
+            )
+            return
+
+        self.log.info("ExtractSlateGlobal is active.")
+
+        # Create dictionary of common data across all slates
+        frame_padding = context.data["anatomy"]["templates"]["defaults"]\
+            ["frame_padding"]
+        slate_common_data = {
+            "@version": str(instance.data["version"]).zfill(frame_padding),
+            "frame_padding": frame_padding,
+            "intent": {"label": "", "value": ""},
+            "comment": "",
+            "scope": "",
+        }
+        slate_common_data.update(instance.data["anatomyData"])
+        if "customData" in instance.data:
+            slate_common_data.update(instance.data["customData"])
```

Comment:
All custom data must be under `"custom"` key in fill data to match other templates for slates and burnins.
To use custom data in template there should be `{custom[my_custom_key]}`. That's to avoid clashes of keys.
```
{
    "project": {"name": "project", "code": "code"},
    "asset": "some asset",
    ...
    "custom": {
        ... custom data. ..
    }
}
```

Reply:
oh ok I see what you meant with your prior comment now, makes sense, I will update!

Reply:
Looking at this better, I'm realizing that all the slate data is already under a `slateGlobal` key on the instance and not flat (`slateGlobal` -> `slate_common_data`). The commented line is only to update the slate common data with any possible `customData` so the suggested change is not equivalent

Having said that, as I'm going to rename the plugin I will also be renaming this key to `htmlSlate`